src/client/stdlib/duckdb.js

```diff
@@ -91,7 +91,7 @@ export class DuckDBClient {
       throw error;
     }
     return {
-      schema: getArrowTableSchema(batch.value),
+      schema: batch.value.schema,
```

Comment:
We’re now deviating from the DatabaseClient specification used in Observable notebooks, but I think it’s better to standardize on the Apache Arrow than our own specification. (Also this schema is unused in Framework — it exists in notebooks to power the data table cell, and we haven’t written than in Framework yet #23, but when we do, it should be based on the Apache Arrow schema.)